### ISSUE TITLE
When using Lambda, the output of the 1/x process is very likely to have nan/inf

### ISSUE THREAD

**Issue by Crystal-yyy** [pytorch/pytorch#76959]
### 🐛 Describe the bug

When using Lambda, the output of the 1/x process is very likely to have nan/inf.  In some cases, the operator will not show output-nan/inf, but will show output-nan/inf. Scenarios, where this happens with lambda, are generally: when the lambda expression carried in the lambda operator has certain risks (the operator does not impose certain constraints on its expression).

**For example：**
structure
`[{'name': 'ReduceMax', 'params': {'keep_dims': True, 'dim': 1, 'tensor_space': 4}}, {'name': 'Upsample2d', 'params': {'scale_factor': 2, 'mode': 'nearest'}}, {'name': 'Lambda', 'params': {'output_shape': [2, 32, 3], 'function': 'lambda x: 1 / x**0.5 + 1.0'}}, {'name': 'Flatten'}, {'name': 'Dense', 'params': {'in_features': 192, 'out_features': 200}}, {'name': 'Softmax'}]`

### Versions

Ubuntu 20.04.2 LTS (GNU/Linux 5.4.0-74-generic x86_64)
Pytorch1.8
gcc (Ubuntu 9.3.0-17ubuntu1~20.04) 9.3.0
NVIDIA-SMI 460.80 Driver Version: 460.80 CUDA Version: 11.2
